build/bin/release-next.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+echo "Release Type?"
+read -rp "> " rtype
+
+echo grunt version:${rtype}
+grunt version:${rtype}
+version=$(./build/bin/version)
```

Comment:
We may want to use something like this here:
```sh
SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
version="$("$SCRIPT_DIR"/version)"
```

This will allow you to use the script outside of the root directory of video.js without issues